corehq/apps/app_manager/tests/data/suite/reports_module_summary_detail.xml

```diff
@@ -31,7 +31,7 @@
         </header>
         <template>
           <text>
-            <xpath function="description"/>
+            <xpath function="report description"/>
```

Comment:
I agree that the UX is pretty poor - one idea I had is to have a checkbox to toggle whether the description is an xpath expression or not.  When it's not set to xpath mode, it would support raw strings and translations.  ```ReportAppConfig``` could then have two separate properties - one a string for xpaths, one a dictionary for translations of ordinary strings.  I'm inclined to let our internal users see the latest changes first and then iterate based on their feedback.